src/main/java/uk/gov/beis/els/service/TemplatePopulator.java

```diff
@@ -79,6 +103,82 @@ public TemplatePopulator setMultilineText(String elementId, String textValue) {
+
+    textValue = textValue.trim();
+
+    int charsPerRow = Integer.parseInt(TemplateUtils.getAttributeByName(TemplateUtils.getSvgElement(template), SVG_MULTILINE_CHARS_PER_ROW_ATTR_NAME));
+    Float maxTextWidth = Float.parseFloat(TemplateUtils.getAttributeByName(line1, "textLength"));
+    Float fontSize = Float.parseFloat(TemplateUtils.getAttributeByName(line1, TEXT_FONT_SIZE_ATTR_NAME));
+    String fontFace = TemplateUtils.getAttributeByName(line1, TEXT_FONT_FACE_NAME);
+    FontProperties fontProperties = FontUtils.FontProperties.fromFontFaceName(fontFace);
+
+    Float estimatedTextWidth = FontUtils.estimatedTextWidth(textValue, fontSize, fontProperties);
+
+    if(estimatedTextWidth < maxTextWidth) {
+      // Text fits on one line without condensing
+      line1.text(""); // clear out row
+      line2.text(textValue);
+      line2.removeAttr("textLength");
+    } else {
+      // Try wrapping using the chars per row specified on the template
+      String wrappedText = WordUtils.wrap(textValue, charsPerRow, "\n", true);
+
+      String [] lines = wrappedText.split("\n");
+
+      Float line1EstimatedWidth = FontUtils.estimatedTextWidth(lines[0], fontSize, fontProperties);
+      Float line2EstimatedWidth = FontUtils.estimatedTextWidth(lines[1], fontSize, fontProperties);
+
+      if(line1EstimatedWidth > maxTextWidth || line2EstimatedWidth > maxTextWidth || lines.length > 2) {
+        // At least one line is too wide, or we have more than 2 lines, so split the text as evenly as possible
+        // across two lines then apply condensing so it fits
+
+        // Wrap text onto lines of about equal length
+        String evenlyWrappedText = WordUtils.wrap(textValue, textValue.length()/2, "\n", true);
+        lines = evenlyWrappedText.split("\n");
+
+        // It might have wrapped onto more than 2 lines, so just make line 2 everything that didn't fit on line 1
+        String line1Text = lines[0];
+        String line2Text = textValue.substring(line1Text.length());
+
+        // Recalculate the estimated widths
+        line1EstimatedWidth = FontUtils.estimatedTextWidth(line1Text, fontSize, fontProperties);
+        line2EstimatedWidth = FontUtils.estimatedTextWidth(line2Text, fontSize, fontProperties);
+
+        // Change the textLength of the shorter line so that the characters are condensed by the same amount on both lines
+        if(line1EstimatedWidth > line2EstimatedWidth) {
+          Float lengthRatio = line2EstimatedWidth / line1EstimatedWidth;
+          line2.attr("textLength", String.valueOf(maxTextWidth * lengthRatio));
+        } else {
+          Float lengthRatio = line1EstimatedWidth / line2EstimatedWidth;
+          line1.attr("textLength", String.valueOf(maxTextWidth * lengthRatio));
+        }
+
+        line1.text(line1Text);
+        line2.text(line2Text);
+      } else {
+        // The text wrapped successfully using the chars per row value
+        if (lines.length == 1) {
```

Comment:
Think this check needs to be moved up in the logic, you're already accessing `lines[1]` in line 132, so this check needs to be done before that point

Reply:
This length check if/else block is also duplicated in `setMultilineText()` so it would be nice to move that out into its own method